api/interestr/website/views.py

```diff
@@ -2,5 +2,19 @@
 from __future__ import unicode_literals
 
 from django.shortcuts import render
+from django.shortcuts import redirect
+from django.contrib.auth import authenticate, login
+
+from django.views import generic
+from django.views.generic.edit import CreateView, UpdateView, DeleteView
+
+from .models import Group
 
 # Create your views here.
+class GroupView(generic.ListView):
+    if not request.user.is_authenticated():
```

Comment:
Lol, this is more wrong that the last one. It's ok I'll just fix this later

Reply:
Really? 😄  but if you think the syntax is the problem i saw [this](https://docs.djangoproject.com/en/1.11/topics/auth/default/#limiting-access-to-logged-in-users) in the documentation. I might be wrong but it should also work for the class views

Reply:
That's a method. You can write code in a method. You cannot write code into a class definition. Have a look at this: https://docs.djangoproject.com/en/1.11/topics/class-based-views/intro/#decorating-the-class

Reply:
Why do you not try your changes in local, so that you can track these errors?